generators/firechip/src/test/scala/ScalaTestSuite.scala

```diff
@@ -125,51 +130,59 @@ abstract class FireSimTestSuite(
   runSuite("verilator")(FastBlockdevTests)
 }
 
-class RocketF1Tests extends FireSimTestSuite("FireSimNoNIC", "DDR3FRFCFSLLC4MB_FireSimRocketChipQuadCoreConfig", "BaseF1Config")
-class BoomF1Tests extends FireSimTestSuite("FireSimNoNIC", "DDR3FRFCFSLLC4MB_FireSimBoomConfig", "BaseF1Config")
-class RocketNICF1Tests extends FireSimTestSuite("FireSim", "DDR3FRFCFSLLC4MB_FireSimRocketChipConfig", "BaseF1Config") {
+class RocketF1Tests extends FireSimTestSuite("FireSim", "DDR3FRFCFSLLC4MB_FireSimQuadRocketConfig", "WithSynthAsserts_BaseF1Config")
+class BoomF1Tests extends FireSimTestSuite("FireSim", "DDR3FRFCFSLLC4MB_FireSimBoomConfig", "BaseF1Config")
+class RocketNICF1Tests extends FireSimTestSuite("FireSim", "WithNIC_DDR3FRFCFSLLC4MB_FireSimRocketConfig", "BaseF1Config") {
   runSuite("verilator")(NICLoopbackTests)
 }
-class RamModelRocketF1Tests extends FireSimTestSuite("FireSimNoNIC", "FireSimRocketChipDualCoreConfig", "BaseF1Config_MCRams")
-class RamModelBoomF1Tests extends FireSimTestSuite("FireSimNoNIC", "FireSimBoomConfig", "BaseF1Config_MCRams")
-
-abstract class FireSimTraceGenTest(targetConfig: String, platformConfig: String)
-    extends firesim.TestSuiteCommon with IsFireSimGeneratorLike {
-  val longName = names.topModuleProject + "." + names.topModuleClass + "." + names.configs
-
-  lazy val generatorArgs = GeneratorArgs(
-    midasFlowKind = "midas",
-    targetDir = "generated-src",
-    topModuleProject = "firesim.firesim",
-    topModuleClass = "FireSimTraceGen",
-    targetConfigProject = "firesim.firesim",
-    targetConfigs = targetConfig ++ "_WithScalaTestFeatures",
-    platformConfigProject = "firesim.firesim",
-    platformConfigs = platformConfig)
-
-  // From HasFireSimGeneratorUtilities
-  // For the firesim utilities to use the same directory as the test suite
-  override lazy val testDir = genDir
-
-  // From TestSuiteCommon
-  val targetTuple = generatorArgs.tupleName
-  val commonMakeArgs = Seq(s"DESIGN=${generatorArgs.topModuleClass}",
-                           s"TARGET_CONFIG=${generatorArgs.targetConfigs}",
-                           s"PLATFORM_CONFIG=${generatorArgs.platformConfigs}")
-
-  it should "pass" in {
-    assert(make("fsim-tracegen") == 0)
-  }
-}
-
-class FireSimLLCTraceGenTest extends FireSimTraceGenTest(
-  "DDR3FRFCFSLLC4MB_FireSimTraceGenConfig", "BaseF1Config")
-
-class FireSimL2TraceGenTest extends FireSimTraceGenTest(
-  "DDR3FRFCFS_FireSimTraceGenL2Config", "BaseF1Config")
+// Disabled until RAM optimizations re-enabled in multiclock
+//class RamModelRocketF1Tests extends FireSimTestSuite("FireSim", "FireSimDualRocketConfig", "BaseF1Config_MCRams")
+//class RamModelBoomF1Tests extends FireSimTestSuite("FireSim", "FireSimBoomConfig", "BaseF1Config_MCRams")
+
+// Multiclock tests
+class RocketMulticlockF1Tests extends FireSimTestSuite(
+  "FireSimMulticlockPOC",
+  "FireSimQuadRocketMulticlockConfig",
+  "WithSynthAsserts_BaseF1Config")
+
+// Jerry broke these -- damn it Jerry.
```

Comment:
Is this already tracked in a PR/Issue?

Reply:
Oops
I don't think this tracegen-specific stuff should be tracked in Firechip anyways. Doing a tracegen simulation in firesim should just require changing the BuildTop key.

Reply:
Yeah i'm in 100% agreement. No one runs the scala tests except me though. I'll fix it when i get the CI to run the scalatests for firechip.